guides/micronaut-security-session/kotlin/src/test/groovy/example/micronaut/SessionAuthenticationSpec.groovy

```diff
@@ -1,79 +0,0 @@
-package example.micronaut
-
-import geb.spock.GebSpec
-import io.micronaut.runtime.server.EmbeddedServer
-import io.micronaut.test.extensions.spock.annotation.MicronautTest
-
-import jakarta.inject.Inject
-
-//tag::clazz[]
-@MicronautTest // <1>
-class SessionAuthenticationSpec extends GebSpec {
```

Comment:
why are we deleting the test?

Reply:
see https://github.com/micronaut-projects/micronaut-guides/pull/456/commits/5201b6470974aebbc750afa71e580014ea8b1a79